docs/home/how-to-use-multiple-templates.md

```diff
@@ -0,0 +1,80 @@
+# How to Use Multiple Templates
```

Comment:
It was noted that this page is not currently included on the docs site.  I just wanted to make sure this was intentional.